docs/source/synthetic_population.rst

```diff
@@ -0,0 +1,87 @@
+
+To fulfill the TAZ requirement, we estimated the total households based on the United Nations `Population Division <https://www.un.org/development/desa/pd/data/household-size-and-composition>`_ information about household composition. As there is no data available for all countries or autonomous regions, for countries that are not present in the sample, we considered their household informations to be similar to their closest neighbors. As there are many data sources available in the Population Division sheet, we kept the most recent data for each country, as long as it had information about the total number of households.
+
+The total number of households in each TAZ is a sum of all households with different sizes. To match the information provided by the UN, we consider households with 1 person (HHBASE1), 2 to 3 persons (HHBASE2), 4 to 5 persons (HHBASE4), and with more than 6 persons (HHBASE6). Hence, we sum all population by sex and age in the zone, divide by the average number of persons in the household, and multiply by the percentage of households of each size expected for the country. Finally, we sum the integer expected values for each household size to obtain the mandatory expected number of households in the TAZ (HHBASE).
+
+As we currently do not have validation data to assess the quality of the synthetic populations created, we use the following variables from the UN Population Division file to validate the population:
+
+- Percentage of households with at least one person under 15 years old;
+- Percentage of households with at least one person under 18 years old;
+- Percentage of households with at least one person under 20 years old;
+- Percentage of households with at least one person over 60 years old;
+- Percentage of households with at least one person over 65 years old;
+- Percentage of households with at least one person under 15 and one person over 60;
+- Percentage of households with at least one person under 15 and one person over 65;
+- Percentage of households with at least one person under 18 and one person over 60;
+- Percentage of households with at least one person under 18 and one person over 65;
+- Percentage of households with at least one person under 20 and one person over 60;
+- Percentage of households with at least one person under 20 and one person over 65.
+
+As part of the validation process, we also run the validation process presented by the development team of PopulationSim, available `here <https://github.com/activitysim/populationSim/tree/master/scripts>`_.
+
```

Comment:
I think it would be helpful to include images of the validations we produce and mention that they are produced alongside with the results (I didn;t see that, but we should create the validation charts as images alongside the results on disk

Reply:
I would put the link to the notebook itself https://github.com/ActivitySim/populationsim/blob/master/scripts/validation.ipynb